pyteal/ast/ternaryexpr_test.py

```diff
@@ -72,6 +72,38 @@ def test_substring_invalid():
         Substring(Bytes("my string"), Int(0), Txn.sender())
 
 
+def test_extract():
```

Comment:
We might want to check for the `Teal4Options` compile error here too?

Reply:
Good catch, I'll add this